src/components/lint/resourcelimits.ts

```diff
@@ -7,6 +7,10 @@ import { warningOn, childSymbols } from './linter.utils';
 // .containers[each].resources.limits.{cpu,memory}
 
 export class ResourceLimitsLinter implements LinterImpl {
+    name(): string {
+        return "resource-limits";
```

Comment:
doesn't need to happen here, but we should document standard linters

Reply:
added doc to the readme.